cmd/otelcol/main.go

```diff
@@ -25,14 +25,14 @@ import (
 )
 
 func main() {
-	handleErr := func(err error) {
+	handleErr := func(message string, err error) {
 		if err != nil {
-			log.Fatalf("Failed to run the service: %v", err)
+			log.Fatalf("%s: %v", message, err)
 		}
 	}
 
 	factories, err := defaults.Components()
-	handleErr(err)
+	handleErr("Failed to build default components", err)
```

Comment:
Here and below - changed fatal error message to be more specific to the stage, as requested by @pjanotti .